app/src/main/java/com/geekbrains/try2/MainActivity.java

```diff
@@ -1,38 +1,55 @@
 package com.geekbrains.try2;
 
+import android.content.Intent;
+import android.net.Uri;
+import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
 import android.widget.Toast;
 
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements Constants{
+
+    private final static int REQUEST_CODE_2_ACTIVITY = 1;
+    private TextView textCity;
 
-    private static final String LIFECYCLE = "Жизненный цикл";
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         makeToast("onCreate");
-    }
+        final TextView textCity = findViewById(R.id.textCity);
```

Comment:
```suggestion
        textCity = findViewById(R.id.textCity);
```
Тут надо использовать поле класса